tutorials/algorithms/03_vqe_simulation_with_noise.ipynb

```diff
@@ -349,57 +351,66 @@
     }
    ],
    "source": [
-    "if counts2 or values2:\n",
-    "    pylab.rcParams['figure.figsize'] = (12, 4)\n",
-    "    pylab.plot(counts2, values2)\n",
-    "    pylab.xlabel('Eval count')\n",
-    "    pylab.ylabel('Energy')\n",
-    "    pylab.title('Convergence with noise, measurement error mitigation enabled')"
+    "if counts or values:\n",
+    "    pylab.rcParams[\"figure.figsize\"] = (12, 4)\n",
+    "    pylab.plot(counts, values)\n",
+    "    pylab.xlabel(\"Eval count\")\n",
+    "    pylab.ylabel(\"Energy\")\n",
+    "    pylab.title(\"Convergence with noise\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Summary\n"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Lets bring the results together here for a summary. \n",
+    "We produced a reference value using a classical algorithm and then proceeded to run VQE on an Aer simulator. While the simulation is ideal (no noise) there is so called shot-noise due to sampling - increasing the number of shots reduces this as more and more samples are taken, but shots were left at the default of 1024 and we see a small effect in the outcome. \n",
     "\n",
-    "We produced a reference value using a classical algorithm and then proceeded to run VQE on a qasm simulator. While the simulation is ideal (no noise) there is so called shot-noise due to sampling - increasing the number of shots reduces this as more and more samples are taken, but shots was left at the default of 1024 and we see a small effect in the outcome. \n",
+    "Then, we added noise using a model taken off a real device and can see the result is affected. \n",
     "\n",
-    "Then we added noise using a model taken off a real device and can see the result is affected. Finally we added measurement noise mitigation which adjusts the results in an attempt to alleviate the affect of noise in the classical equipment measuring the qubits."
+    "In the future, we could apply error mitigation techniques to alleviate the effect of noise in simulation."
```

Comment:
This future statement only relates to possibility of adding mitigation support to Are primitives right. Maybe we should note that if you are using VQE with the runtime then the Estimator there offers noise mitigation via the resilience_level and to see https://qiskit.org/documentation/partners/qiskit_ibm_runtime/ for more details?